match/lib/match/importer.rb

```diff
@@ -149,9 +149,9 @@ def ensure_valid_file_path(file_path, file_description, file_extension, optional
     end
 
     def api_token(params)
-      @api_token ||= Spaceship::ConnectAPI::Token.create(params[:api_key]) if params[:api_key]
-      @api_token ||= Spaceship::ConnectAPI::Token.from_json_file(params[:api_key_path]) if params[:api_key_path]
-      return @api_token
+      api_token = Spaceship::ConnectAPI::Token.from(hash: params[:api_key], filepath: params[:api_key_path])
```

Comment:
in this class (and a few below) we are not consistent with the way we log about wether we reuse or create a token.

For example the implementation in `fastlane/lib/fastlane/actions/set_changelog.rb` treats them differently

Reply:
`match` has a bunch of tools that are only called from the CLI as sub commands. So this one should never really have token to reuse already 🤷‍♂️  But I can make these look and reuse a token if there is an existing one!

Reply:
lol, I read your original comment wrong 😛  I will make this change!